exporters/pelorus/timeutil.py

```diff
@@ -62,6 +62,26 @@ def parse_guessing_timezone_DYNAMIC(timestring: str, format: str) -> datetime:
         return parsed.replace(tzinfo=timezone.utc)
 
 
+def to_epoch_from_string(
+    timestring: str, allowed_length: list[int] = [10, 13]
```

Comment:
Any reason to have customizable lengths?

Reply:
just to make function nicer :)

Reply:
I don't understand, what would it mean if someone passed something like `[5]`? It's not like we enforce leading zeroes-- it's just to determine milliseconds versus seconds, right?

Reply:
I am trying to ensure some annotation is not consisting of numbers that are 6,7,8,9,22 in length see couple of examples:

```
>>> from datetime import datetime
>>> datetime.fromtimestamp(int("123"))
datetime.datetime(1970, 1, 1, 1, 2, 3)
>>> datetime.fromtimestamp(int("1234567"))
datetime.datetime(1970, 1, 15, 7, 56, 7)
>>> datetime.fromtimestamp(int("123456789012"))
datetime.datetime(5882, 3, 11, 1, 30, 12)
>>> datetime.fromtimestamp(int("00000000000013900000434"))
datetime.datetime(2410, 6, 22, 17, 13, 54)
>>> datetime.fromtimestamp(int("00000000000001600000434"))
datetime.datetime(2020, 9, 13, 14, 33, 54)
```

So we are limiting here stupid human errors a bit. See the last example which happens to be quite fine date...

Reply:
actually we should allow only seconds, see:

```
>>> datetime.fromtimestamp(int("1663772656490"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: year 54692 is out of range
```